modules/Microsoft.VirtualMachineImages/imageTemplates/deploy.bicep

```
@@ -24,6 +24,11 @@ param osDiskSizeGB int = 128
 @description('Optional. Resource ID of an already existing subnet, e.g. \'/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/virtualNetworks/<vnetName>/subnets/<subnetName>\'. If no value is provided, a new VNET will be created in the target Resource Group.')
 param subnetId string = ''
 
+@description('''Optional. List of User-Assigned Identities associated to the Build VM for accessing Azure resources such as Key Vaults from your customizer scripts.
+Be aware, the user assigned identity for Azure Image Builder must have the "Managed Identity Operator" role assignment on all the user assigned identities for Azure Image Builder to be able to associate them to the build VM.
+''')
+param vmUserAssignedIdentities array = []
```

I'd suggest to keep the parameter name close to the resource reference, so
```suggestion
param userAssignedIdentities array = []
```

---

Note, the references would need to be manually updated too, if the suggestion is applied

---

updated